pom.xml

```diff
@@ -5,13 +5,17 @@
 	<parent>
 		<groupId>org.jenkins-ci.plugins</groupId>
 		<artifactId>plugin</artifactId>
-		<version>1.565.3</version>
+		<version>1.642.3</version>
```

Comment:
Note: I fixed that locally,  but not using a 2.x pom would make the  `jenkins.version` usage below ignored.